# HG changeset patch
# User Thomas Weber <thomas.weber.mail@gmail.com>
# Date 1220978291 14400
#      Tue Sep 09 12:38:11 2008 -0400
# Branch release-3-0-x
# Node ID 5860aaef2f6fda163ac4fbd97141b8f37c30867a
# Parent  806c1e8a07c8374b7c8b7203dd1199daa8464509
typo in comma separated lists chapter

diff --git a/doc/interpreter/container.txi b/doc/interpreter/container.txi
--- a/doc/interpreter/container.txi
+++ b/doc/interpreter/container.txi
@@ -712,17 +712,17 @@ example
 
 @example
 [@var{i}, @var{j}] = ceil (find (@var{x}, [], "last"));
 @end example
 
 @noindent
 where @code{@var{i}, @var{j}} is equally a comma separated list. Comma
 separated lists can not be directly manipulated by the user. However,
-both structures are cell arrays can be converted into comma
+both structures and cell arrays can be converted into comma
 separated lists, which makes them useful to keep the input arguments and
 return values of functions organized. Another example of where a comma
 separated list can be used is in the creation of a new array. If all the
 accessed elements of a cell array are scalars or column vectors, they
 can be concatenated into a new column vector containing the elements, by
 surrounding the list with @code{[} and @code{]} as in the following
 example
 
